src/overlays/actors/ovl_En_Nimotsu/z_en_nimotsu.c

```diff
@@ -5,6 +5,8 @@
  */
 
 #include "z_en_nimotsu.h"
+#include "assets/objects/object_boj/object_boj.h"
+#include "macros.h"
```

Comment:
The `macros.h` header should already be included by the `global.h` header, which is included by this actor header, so you can remove it safely